mPowerSDK/TasksAndSteps/APHActivityManager.m

```diff
@@ -219,14 +221,31 @@ - (ORKOrderedTask *)createCustomWalkingTask
+    ORKInstructionStep *instructionStep2 = (ORKInstructionStep *)orkTask.steps[1];
+    [instructionStep2 setText:NSLocalizedStringWithDefaultValue(@"APH_WALKING_DESCRIPTION_2", nil, APHLocaleBundle(), @"\u2022 Please wear a comfortable pair of walking shoes and find a flat, smooth surface for walking.\n\n\u2022 Try to walk continuously by turning at the ends of your path, as if you are walking around a cone.\n\n\u2022 Importantly, walk at your normal pace. You do not need to walk faster than usual.", @"Detailed instructions for performing walking activity.")];
+    [instructionStep2 setDetailText:NSLocalizedStringWithDefaultValue(@"APH_WALKING_DESCRIPTION_2_DETAIL", nil, APHLocaleBundle(), @"Put your phone in a pocket or bag and follow the audio instructions.", @"Instructions for how to proceed to active step of walking activity.")];
     
-    ORKActiveStep *activeStep = (ORKActiveStep *)orkTask.steps[5];
-    [activeStep setTitle:titleString];
-    [activeStep setSpokenInstruction:spokenInstructionString];
+    NSString  *titleFormatWalk = NSLocalizedStringWithDefaultValue(@"APH_WALKING_WALK_TEXT_INSTRUCTION", nil, APHLocaleBundle(), @"Walk back and forth for %@ seconds.", @"Written instructions for the walking step of the walking activity, to be filled in with the number of seconds to walk.");
+    NSString  *titleStringWalk = [NSString stringWithFormat:titleFormatWalk, APHLocalizedStringFromNumber(@(kStandStillDuration))];
+    NSString  *spokenInstructionFormatWalk = NSLocalizedStringWithDefaultValue(@"APH_WALKING_WALK_SPOKEN_INSTRUCTION", nil, APHLocaleBundle(), @"Walk back and forth for %@ seconds.", @"Spoken instructions for the walking step of the walking activity, to be filled in with the number of seconds to walk.");
+    NSString  *spokenInstructionStringWalk = [NSString stringWithFormat:spokenInstructionFormatWalk, APHLocalizedStringFromNumber(@(kStandStillDuration))];
+    
+    ORKWalkingTaskStep *walkStep = (ORKWalkingTaskStep *)orkTask.steps[3];
+    [walkStep setTitle:titleStringWalk];
+    [walkStep setSpokenInstruction:spokenInstructionStringWalk];
+    // we want to ignore steps and just go for 30 seconds, but can't disable step counting without disabling the
+    // pedometer, so we've set way more steps than 30 seconds' worth, and here we'll to set the stepDuration
+    // fallback value manually to 30 seconds.
+    walkStep.stepDuration = kWalkDuration;
+    
+    NSString  *titleFormatStand = NSLocalizedStringWithDefaultValue(@"APH_WALKING_STAND_STILL_TEXT_INSTRUCTION", nil, APHLocaleBundle(), @"Turn around 360 degrees, then stand still, with your feet about shoulder-width apart. Rest your arms at your side and try to avoid moving for %@ seconds.", @"Written instructions for the standing-still step of the walking activity, to be filled in with the number of seconds to stand still.");
```

Comment:
From @trister (see his comment on https://sagebionetworks.jira.com/browse/BRIDGE-637), who I believe got it directly from the corresponding task in the Harvard FPHS app (which we are moving this task toward).